avocado/core/varianter.py

```diff
@@ -209,11 +209,6 @@ def _process_default_params(self, args):
             default_params.merge(default_param)
         self._default_params = default_params
         self.default_params.clear()     # We don't need these anymore
-        # FIXME: Backward compatibility params, to be removed when 36 LTS is
-        # discontinued
-        if (not getattr(args, "variants-skip-defaults", False) and
-                hasattr(args, "default_avocado_params")):
-            self._default_params.merge(args.default_avocado_params)
```

Comment:
The same applies here, please consider embedding this code into `parse` as without the compatibility code it's just few lines related to parsing.